File: Java/String/program20.java
//Accept the string from user and display that string in uppercase

import java.util.*;

class Stringx
{
    public String str;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the string");
        str = sobj.nextLine();
    }

    public void struprX()
    {
        System.out.print("String in upper case : "+str.toUpperCase());        
    }
}

class program20
{
    public static void main(String arg[])
    {
        Stringx obj = new Stringx();

        obj.Accept();

        obj.struprX();
    }
}

File: C/String/program20.c
//Accept the string from user and reverse that string

#include<stdio.h>

void strrevX(char str[])
{
    char *start = str;
    char *end = str;
    char Temp = '\0';

    while(*end != '\0')
    {
        end++;
    }
    end--;
    
    while(start <= end)
    {
        Temp = *start;
        *start = *end;
        *end = Temp;

        start++;
        end--;
    }
}

int main()
{
    char Arr[20];

    printf("Enter string: \n");
    scanf("%[^'\n']s",Arr);

    strrevX(Arr);
    printf("String after reverse : %s ",Arr);

    return 0;
}
